test/unit_tests/test_metadata_scheme_file.py

```diff
@@ -85,203 +87,263 @@ def setUp(self):
+        self.assertTrue(emsg in str(context.exception))
+        self.assertTrue("3 errors found comparing" in str(context.exception))
 
-    def test_CCPPeq1_var_missing_in_meta(self):
-        """Test correct detection of a variable that IS PRESENT the subroutine argument list
-           (due to a pre-processor directive: #ifdef CCPP), and IS NOT PRESENT in meta file"""
-        #Setup
-        scheme_files = [os.path.join(self._sample_files_dir, "CCPPeq1_var_missing_in_meta.meta")]
-        #Exercise
-        with self.assertRaises(Exception) as context:
-            parse_scheme_files(scheme_files, self._run_env_ccpp)
-        #Verify 3 correct error messages returned
-        self.assertTrue('Variable mismatch in CCPPeq1_var_missing_in_meta_finalize, variables missing from metadata header.'
-                         in str(context.exception))
-        self.assertTrue('Out of order argument, errmsg in CCPPeq1_var_missing_in_meta_finalize' in str(context.exception))
-        self.assertTrue('Out of order argument, errflg in CCPPeq1_var_missing_in_meta_finalize' in str(context.exception))
-        self.assertTrue('3 errors found comparing' in str(context.exception))
+    def test_ccpp_eq1_var_missing_in_meta(self):
+        """Test correct detection of a variable that
+           IS PRESENT the subroutine argument list
+           (due to a pre-processor directive: #ifdef CCPP),
+            and IS NOT PRESENT in meta file"""
+        # Setup
+        scheme_files = [os.path.join(self._sample_files_dir,
+                                     "CCPPeq1_var_missing_in_meta.meta")]
+        # Exercise
+        with self.assertRaises(CCPPError) as context:
+             _, _ = parse_scheme_files(scheme_files, self._run_env_ccpp)
+        # Verify 3 correct error messages returned
+        emsg = "Variable mismatch in CCPPeq1_var_missing_in_meta_finalize, "+ \
+            "variables missing from metadata header."
+        self.assertTrue(emsg in str(context.exception))
+        emsg = "Out of order argument, errmsg in CCPPeq1_var_missing_in_meta_finalize"
+        self.assertTrue(emsg in str(context.exception))
+        emsg = "Out of order argument, errflg in CCPPeq1_var_missing_in_meta_finalize"
+        self.assertTrue(emsg in str(context.exception))
+        self.assertTrue("3 errors found comparing" in str(context.exception))
 
-# pylint: enable=invalid-name
+    def test_scheme_ddt_only(self):
+        """Test correct detection of a "scheme" file which contains only
+           DDT definitions"""
+        # Setup
+        scheme_files = [os.path.join(self._host_files_dir, "ddt2.meta")]
+        # Exercise
+        scheme_headers, table_dict = parse_scheme_files(scheme_files,
+                                                        self._run_env_ccpp)
+#        with self.assertRaises(CCPPError) as context:
+#             _, _ = parse_scheme_files(scheme_files, self._run_env_ccpp)
+#        # Verify correct error messages returned
```

Comment:
Can this be deleted since it's commented out?

Reply:
deleted